python/tests/test_genotypes.py

```diff
@@ -2100,3 +2102,60 @@ def test_variant_frequencies_no_samples(self, ts_fixture, caplog):
             freqs = variant.frequencies()
             assert caplog.text.count("frequencies undefined") == 1
         assert np.all(np.isnan(list(freqs.values())))
+
+    def test_variant_str(self):
+        """
+        Test using a simple dummy tree sequence for testing.
+        It has only one tree and one site, whose variant has the alleles
+        ('A', 'T', 'G', None).
+        """
+        tables = tskit.TableCollection(10)
+        tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0)
+        tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0)
+        tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0)
+        tables.sites.add_row(position=5, ancestral_state="A")
+        tables.mutations.add_row(site=0, node=0, derived_state="T")
+        tables.mutations.add_row(site=0, node=1, derived_state="G")
```

Comment:
Can we put one of the derived states to a unicode character, e.g. the classic 💩 ?!

Reply:
Oh, and set one of the others to '' (empty string) and other to multiple characters, e.g. `AAC`

Reply:
The poop emoji messes up the alignment of the columns...
<img width="325" alt="Screen Shot 2022-07-27 at 4 23 41 PM" src="https://user-images.githubusercontent.com/5580375/181286371-91c84887-21bf-43f6-aad0-629c87af95b1.png">

Reply:
It is because the poop emoji is of the same width as the other characters.

Reply:
I don't think we care. This is an edge case. But it's worth keeping a weird character in the tests to check that we don't bug out when trying to print, etc.